README.md

```diff
@@ -0,0 +1,74 @@
+![image](https://img.shields.io/pypi/dm/discord-py-slash-command.svg)
+![image](https://img.shields.io/pypi/pyversions/discord-py-interactions.svg)
+![image](https://img.shields.io/pypi/v/discord-py-interactions.svg)
+![image](https://readthedocs.org/projects/interactionspy/badge/?version=latest)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+![image](https://discord.com/api/guilds/789032594456576001/embed.png)
+
+------------------------------------------------------------------------
+
+# Interactions.py - A Feature-rich Discord Bot Framework for Python
+
+A highly extensible, easy to use, and feature complete bot framework for Discord, `Interactions.py` is the culmination of years of experience in the Discord API and bot development. This library has been built from the ground up with community feedback and suggestions in mind, providing a modern and intuitive API for easy interaction with the Discord API in Python.
```

Comment:
```suggestion
A highly extensible, easy to use, and feature complete client library for Discord.

interactions.py is the culmination of years of experience with Discord's APIs and bot development. This library has been built from the ground up with community feedback and suggestions in mind. This library provides a modern and intuitive set of language bindings for easy interaction with Discord.
```

Reply:
Bot framework was used to match venacular used by other libraries. Happy to drop the "bot" but I think "framework" should remain